src/Compilers/VisualBasic/BasicAnalyzerDriver/VisualBasicDeclarationComputer.vb

```diff
@@ -119,35 +139,48 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         For Each decl In typeBlock.Members
                             ComputeDeclarationsCore(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken)
                         Next
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken))
+                        Dim attributes = GetAttributes(typeBlock.BlockStatement.AttributeLists)
```

Comment:
>Dim attributes = GetAttributes(typeBlock.BlockStatement.AttributeLists) [](start = 24, length = 71)

Aren't we doing this below?

Reply:
[https://github.com/dotnet/roslyn/issues/23347](https://github.com/dotnet/roslyn/issues/23347)

---
In reply to: [152419048](https://github.com/dotnet/roslyn/pull/23318#discussion_r152419048) [](ancestors = 152419048)